Sources/RInAppMessaging/Extensions/UIColor+IAM.swift

```diff
@@ -11,6 +16,8 @@ internal extension UIColor {
         }
     }
 
+    static let buttonBorderDefaultColor = UIColor(hexString: "#D1D1D1")!
```

Comment:
I don't think so. It's a constant. If it doesn't crash now, it shouldn't crash later. Having `buttonBorderDefaultColor` as optional would be misleading in my opinion. But I could add a nil coalescence with `UIColor.grey` for example just to avoid that force unwrap.